lib/quicly.c

```diff
@@ -3873,29 +3873,50 @@ size_t quicly_send_retry(quicly_context_t *ctx, ptls_aead_context_t *token_encry
     return ret == 0 ? buf.off : SIZE_MAX;
 }
 
-static int send_handshake_flow(quicly_conn_t *conn, size_t epoch, quicly_send_context_t *s, int ack_only, int send_probe)
+static struct st_quicly_pn_space_t *setup_send_epoch(quicly_conn_t *conn, size_t epoch, quicly_send_context_t *s)
 {
     struct st_quicly_pn_space_t *ack_space = NULL;
```

Comment:
maybe call this send_pn_space? or send_space?

Reply:
Sounds good. I've changed the name to `setup_send_space`, as we often omit "pn_" when that can be implied.